helm/examples/azure/values-retained-azurefile.yaml

```diff
@@ -36,6 +36,11 @@ storage:
     labels:
     - data # NB: if not set, this would be the default
 
+  - name: mydropzone
```

Comment:
I'm not quite sure why these are chcked in - since it matches the contents of the generated NOTES.txt

Reply:
do you mean, why are any of the examples/*/values*.yaml files checked in, not just this azure one, which as you say are version of what would be generated from NOTES.txt ?

Not sure either, or if clearer, the README.md should explain what they are, but we could do without them, and instruct people to capture and use the output of NOTES.txt.

Reply:
I meant the 3 examples that are generated from notes.txt.  Doesn't really matter though